config/jobs/prom-rule-reloader/prom-rule-reloader-presubmits.yaml

```diff
@@ -0,0 +1,73 @@
+job_template: &job_template
+  max_concurrency: 10
+  path_alias: github.com/pusher/prom-rule-reloader
+  agent: kubernetes
+  always_run: true
+  skip_report: false
+  decorate: true
+
+container_template: &container_template
+  image: quay.io/pusher/golang-builder:v20190404-3ebb4c5
+  name: runner
+  command: ["/usr/local/bin/runner"]
+
+container_template_small: &container_template_small
+  <<: *container_template
+  resources:
+    requests:
+      cpu: 1
+      memory: 1Gi
+    limits:
+      cpu: 2
+      memory: 2Gi
+
+presubmits:
+  pusher/prom-rule-reloader:
+    - name: pull-prom-rule-reloader-lint
+      <<: *job_template
+      spec:
+        containers:
+          - <<: *container_template_small
+            args:
+              - ./configure && make fmt vet lint
+      trigger: "(?m)^/test (?:.*? )?(lint|all)(?:.*? )?$"
+      rerun_command: "/test lint"
+
+    - name: pull-prom-rule-reloader-build
+      <<: *job_template
+      spec:
+        containers:
+          - <<: *container_template_small
+            args:
+              - ./configure && make build
+      trigger: "(?m)^/test (?:.*? )?(build|all)(?:.*? )?$"
+      rerun_command: "/test build"
+
+    - name: pull-prom-rule-reloader-tests
+      <<: *job_template
+      spec:
+        containers:
+          - <<: *container_template_small
+            args:
+              - ./configure && make test
+      trigger: "(?m)^/test (?:.*? )?(tests|all)(?: .*?)?$"
+      rerun_command: "/test tests"
+
+    - name: pull-prom-rule-reloader-build-docker
+      <<: *job_template
+      always_run: false
+      labels:
+        preset-dind-enabled: "true"
+        preset-quay-credentials: "true"
+      spec:
+        containers:
+          - <<: *container_template
+            args:
+              - ./configure;
```

Comment:
Is the `configure` script required for running the docker steps?

Reply:
Probably not, but doesn't exactly hurt, either, does it?

Reply:
True, just wondering if the output it prints is useful to capture or not when it's unnecessary? I'm like 2/10 on this so 🤷‍♂️